hudi/manifest.json

```diff
@@ -0,0 +1,32 @@
+{
+  "display_name": "Hudi",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "hudi",
+  "metric_prefix": "hudi.",
+  "metric_to_check": "hudi.action.duration",
+  "creates_events": false,
+  "short_description": "Track metrics for your Hudi configuration.",
+  "guid": "abfa7624-ac5d-4601-b96c-03a2243ff7c7",
+  "support": "core",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Hudi",
+  "type": "check",
+  "is_public": false,
```

Comment:
So this integration won't be publicly available once it's released?

Reply:
Yes, we do this with all new integrations and make them available once the new agent is released. For hudi in particular we will wait to make it public it until `0.10.0` is released.